### DISCUSSION TITLE
Tow questions : Notebook wide search and How to Publish Desktop Wiki

### DISCUSSION THREAD

**Discussion by seri3s** [zim-desktop-wiki/zim-desktop-wiki#2437] [Q&A]
Hi,

I am using ZIM-0.73.5.
There is no Notebook wide search but search on a page.
I guess I need Notebook wide search function but is it in  a higher versions?

I found that zim-wiki.org made of zim itself.
(It says that "This website was written in Zim !")

How was that done?

Thanks and Regards,

**Comment by bruvellu** [comment]
Hello,

- You can search notebook-wide with `Search` > `Search...` in the menu (or press CTRL+SHIFT+F).
- To make a webpage, export your notebook as HTML following the instructions at `File` > `Export`.

These and other functions are documented in `Help` > `Contents` (or press F1).

Cheers

**Comment by seri3s** [comment]
Thanks for your response.

I know that search function but it searches only in the current page.
If I am looking at home and search something on subsidiary pages, it can
not show.

I know export function and I did it but links of some URL and image are
internal which are not working on internet.

I guess zim-wiki.org is on DocumentRoot of some Web-server itself then
export all in html format with zero-five-eight template.

Regards,



2023年7月29日(土) 17:07 Bruno C. Vellutini ***@***.***>:

**Reply by introt** [reply]
The search they're referring to is the one that pops open another window and should indeed search notebook wide - unless you tick the "Limit search to the current page and its subpages" box. You may also try the Instant Search plugin: e3rd/zim-plugin-instantsearch
